conda/common/logic.py

```diff
@@ -838,6 +838,8 @@ def sum_val(sol, odict):
                 if newsol is None:
                     lo = mid + 1
                     log.trace("Bisection failure, new range=(%d,%d)" % (lo, hi))
+                    if lo > hi:
```

Comment:
Huh, this should "never" happen.
Just breaking out of the loop might violate some invariant/expected post-condition, though.
We have to rather fix the cause of this. I'll try to have a look at this this week.
@msarahan, do you have a failing test case for this? That might make debugging considerably faster for me ;).